src/test/javascript/spec/component/course/course-overview.service.spec.ts

```diff
@@ -47,18 +51,37 @@ describe('CourseOverviewService', () => {
         pastExercise.releaseDate = lastWeek;
         pastExercise.title = 'Past Exercise';
 
+        course.id = 2;
```

Comment:
There is no reason to change the course id before the creation of the exercises. Since the exercises store a reference to the course only the last id change actually has an impact

Reply:
done